packages/common/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@latticexyz/common",
+  "version": "1.42.0",
+  "description": "Common low level logic shared between packages",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/latticexyz/mud.git",
+    "directory": "packages/common"
+  },
+  "license": "MIT",
+  "type": "module",
+  "exports": {
+    "./codegen": "./dist/codegen/index.js",
+    "./foundry": "./dist/foundry/index.js"
+  },
+  "typesVersions": {
+    "*": {
+      "codegen": [
+        "./src/codegen/index.ts"
+      ],
+      "foundry": [
+        "./src/foundry/index.ts"
+      ]
+    }
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "vite build",
+    "release": "npm publish --access=public",
```

Comment:
I noticed this sprinkled everywhere - is this needed?

Reply:
you mean the `release` script? It's used by `release:manual` in the root package.json, which is useful if a lerna release fails midway (after increasing all versions etc) to just push the current state to npm

Reply:
in that case (assuming all `release` scripts are the same in each package), we can just recursively exec this same command but from the root package